project/AppDependencies.scala

```diff
@@ -3,16 +3,16 @@ import play.core.PlayVersion.current
 
 object AppDependencies {
 
-  private val bootstrapPlayVersion = "7.16.0"
+  private val bootstrapPlayVersion = "7.19.0"
   private val hmrcMongoPlayVersion = "0.74.0"
 
   private lazy val compile: Seq[ModuleID] = Seq(
     "commons-validator"            % "commons-validator"           % "1.7",
-    "uk.gov.hmrc"                  %% "play-frontend-hmrc"         % "7.13.0-play-28",
+    "uk.gov.hmrc"                  %% "play-frontend-hmrc"         % "7.14.0-play-28",
     "uk.gov.hmrc"                  %% "bootstrap-frontend-play-28" % bootstrapPlayVersion,
     "uk.gov.hmrc"                  %% "http-caching-client"        % "10.0.0-play-28",
     "uk.gov.hmrc.mongo"            %% "hmrc-mongo-play-28"         % hmrcMongoPlayVersion,
-    "commons-codec"                % "commons-codec"               % "1.15",
+    "commons-codec"                % "commons-codec"               % "20041127.091804",
```

Comment:
latest version is 1.16 please check maven repository https://mvnrepository.com/artifact/commons-codec/commons-codec

Reply:
I tried with that one but failed to update the sbt